geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/formats/GeoJsonExporter.scala

```diff
@@ -40,10 +42,12 @@ class GeoJsonExporter(os: OutputStream, counter: ByteCounter) extends ByteCounte
       if (serializer != null) {
         serializer.endFeatureCollection(writer)
         writer.flush()
-        os.write('\n')
+        stream.os.write('\n')
       }
     } finally {
-      writer.close() // also closes underlying writer and output stream
+      if (writer != null) {
```

Comment:
Hmm... I wonder how the writer could been cleared....  The check is probably sensibly though.

Reply:
if the exporter is never `start`ed, then the writer will be null